assets/js/tabs.js

```diff
@@ -15,19 +15,24 @@
   var interval = 3000;
   var tabCarousel = setInterval(nextSlide, interval);
 
-  $('.tab-content.tab-focus, .nav-tabs.nav-tabs-focus').hover(function(e){
+  $(document).on({
+    mouseenter: function () {
       clearInterval(tabCarousel);
-  }, function(e){
+    },
+    mouseleave: function () {
       tabCarousel = setInterval( nextSlide, interval);
-  });
+    }
+  }, ".tab-content.tab-focus, .nav-tabs.nav-tabs-focus");
 
   function nextSlide() {
-    var tabs = $('.nav-tabs-focus.nav-tabs > li'),
-        active = tabs.filter('.active'),
-        next = active.next('li'),
-        toClick = next.length ? next.find('a') : tabs.eq(0).find('a');
+    if ($('.nav-tabs-focus.nav-tabs > li').length) {
```

Comment:
would be nicer to move this into a variable outside the `nextSlide` function. Do we have case where new tab is added dynamically ?